docs/INDEX.md

```diff
@@ -0,0 +1,63 @@
+# Interview Assignment - Coop Logistics API
+
+**Implement Server side part that can handle client requests
+with specific business logic.**
+
+## Acceptance Criteria
+
+- All three parts must be done.
+- API must be sustainable to handle requests without dropping them.
+
+## Assignment Appendix
+
+Coop Logistics Engine simulates several cargo units that
+deliver goods between different warehouses in the world.
+
+It sends `gRPC` requests to `127.0.0.1:50051`,
+look at service in
+[docker-compose](../docker-compose.yml) "interview_backend_coop_logistics"
+
+## Assignment
+
+The result of this task should be an API system that fulfils the criteria
+described in three parts below. Here is the link to the API documentation
+[Swagger documentation](../api/logistics.swagger.json)
+or [Protofile](../api/v1/logistics.proto) with descriptions.
```

Comment:
Minor: This line about the API specs is probably better suited in the "Part One" section, but as it's already mentioned there so one could probably remove it here.

Also, as the client expects a gRPC endpoint, so it's perhaps not worth mentioning the OpenAPI v2 spec until it's required.

Reply:
Yeah let's do it like that, when we will require to implement it as HTTP server like REST then we can return it back. Will refactor it to be more generic, like here is a folder with API specs.